diagnostic.md

```diff
@@ -5,20 +5,28 @@ Record your responses inside the fenced code blocks below each question.
 1.  Give an example of a visual hierarchy that could be modeled with components.
 
     ```md
-    <!-- your response here -->
+      
+    (https://cloud.githubusercontent.com/assets/388761/12339386/dc1cc062-bae2-11e5-85be-ae33da715b2c.png)
```

Comment:
For this question we were looking for a written description of a visual hierarchy modeled via components.  Any pair of UI elements that are nested are a good example.  ie., a navbar w/ buttons, the sidebar of a blog w/ comments, a list w/ items... etc.